File: content/code-examples/tested/javascript/driver/time-series/sample-app.snippet.example.js
const { MongoClient } = require('mongodb');

// Replace the placeholder with your connection string.
const uri = '<connection-string>';
const client = new MongoClient(uri);

export async function runApp() {
  try {
    // start example code here
    // end example code here
  } finally {
    await client.close();
  }
}

runApp().catch(console.dir);

File: content/code-examples/tested/csharp/driver/TimeSeries/SampleApp.snippet.example.cs
using MongoDB.Driver;

// Replace the placeholder with your connection string.
var uri = "<connection string>";

try
{
    var client = new MongoClient(uri);
    // start example code here

    // end example code here
}
catch (MongoException me)
{
    Console.Error.WriteLine(me.Message);
}
